Reqnroll/ITestRunner.cs

```diff
@@ -1,17 +1,19 @@
+using System;
 using System.Threading.Tasks;
 
 namespace Reqnroll
 {
     public interface ITestRunner
     {
         /// <summary>
-        /// The ID of the parallel test worker processing the current scenario. How the worker ID is obtained is dependent on the test execution framework.
+        /// The ID of the parallel test worker processing the current scenario.
         /// </summary>
         string TestWorkerId { get; }
         FeatureContext FeatureContext { get; }
         ScenarioContext ScenarioContext { get; }
         ITestThreadContext TestThreadContext { get; }
 
+        [Obsolete("TestWorkerId is now managed by Reqnroll internally - Method will be removed in v3")]
```

Comment:
If this PR is merged we should probably note this down somewhere, maybe https://github.com/orgs/reqnroll/projects/3  with a v3 milestone?

Reply:
I did (https://github.com/orgs/reqnroll/projects/3/views/1?pane=issue&itemId=65349394), but since this is just a "draft task", I cannot assign a milestone to it.

Reply:
ah ye, that issue :)